pkg/substitution/substitution.go

```diff
@@ -50,7 +50,7 @@ var intIndexRegex = regexp.MustCompile(intIndex)
 
 // ValidateVariable makes sure all variables in the provided string are known
 func ValidateVariable(name, value, prefix, locationName, path string, vars sets.String) *apis.FieldError {
-	if vs, present, _ := extractVariablesFromString(value, prefix); present {
+	if vs, present, _ := ExtractVariablesFromString(value, prefix); present {
```

Comment:
why is this being exported? I don't see any additional usages of it

Reply:
It might be because I split it up. I will undo it so that I can fix it in the other PR where it is actually used.

Reply:
done.